lib/auth/trust/trustv1/service.go

```diff
@@ -229,6 +238,8 @@ func (s *Service) GenerateHostCert(
 		return nil, trace.Wrap(err)
 	}
 
+	// TODO (Joerger): in v16.0.0, this endpoint should require admin action authorization.
```

Comment:
Why do we have to wait for v16? What is the general plan?

Reply:
This endpoint was added in a minor version of v14, so to keep compatibility with v14.0 clients that still use the http endpoint, we can't require MFA until v16

Reply:
Ah, ok, I see. Makes sense.

Reply:
Couldn't we require MFA in the grpc endpoint but not in the http one? Or would that be too inconsistent?